app/controllers/search_controller.rb

```diff
@@ -58,7 +58,7 @@ def pattern_search
 
   def site_google_search(pattern)
     if pattern.blank?
-      redirect_to(:root)
+      redirect_to observations_path
```

Comment:
See above.

Reply:
LOL! 

I had interpreted your earlier comments to mean that you had worked out a compromise that was not what you wanted.
Looks like it was working as intended though, and this PR is irrelevant.